components/nav-pagination/nav-pagination.js

```diff
@@ -0,0 +1,29 @@
+const pageSelector = document.querySelector('[data-js="numberFrom"]');
+const pagesSelector = document.querySelector('[data-js="numberTo"]');
+let page = 1;
+
+function refreshPaginationIndex(target, pages) {
+  target.addEventListener('click', event => {
+    if (target.getAttribute('data-js') == 'button-prev') {
+      if (page != 1) {
+        page--;
+        setPaginationIndex(page, pages);
+      }
+    } else if (!(page > pages)) {
```

Comment:
if page = pages, then page will increase and get greater than pages (greater than max).
fix suggestion: page => pages
:p